### PR TITLE
Add Prometheus 1.0 blog post

### PR DESCRIPTION
**Pull request by fabxc**

### CODE REVIEW

**Comment by brian-brazil** [comment]
:+1:

File: content/blog/2016-07-18-prometheus-1-0-released.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Absolutely not. We have a long roadmap ahead of us, full of great features to implement. Prometheus will not stay in 1.x for years to come. The infrastructure space is evolving rapidly and we fully intend for Prometheus to evolve with it.
+This means that we will remain willing to question what we did in the past and are open to leave behind things that have lost relevance. There will be new major versions of Prometheus to facilitate future plans like persistent long-term storage, newer iterations of Alertmanager, internal storage improvements, and many things we don’t even know about yet.
+
+## Closing thoughts
+
+We want to thank our fantastic community for field testing new versions, filing bug reports, contributing code, helping out other community members, and shaping Prometheus by participating in countless productive discussions.
+In the end, you are the ones who make Prometheus successful.
+
+Thank you, and keep up the great work!
+
```
Comment (juliusv) [inline content/blog/2016-07-18-prometheus-1-0-released.md]: extra empty line at end :P

File: content/blog/2016-07-18-prometheus-1-0-released.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+author_name: Fabian Reinartz on behalf of the Prometheus team
+---
+
+In January, we published a blog post on [Prometheus’s first year of public existence](https://prometheus.io/blog/2016/01/26/one-year-of-open-prometheus-development/), summarizing what has been an amazing journey for us, and hopefully an innovative and useful monitoring solution for you.
+Since then, [Prometheus has also joined the Cloud Native Computing Foundation](https://prometheus.io/blog/2016/05/09/prometheus-to-join-the-cloud-native-computing-foundation/), where we are in good company, as the second charter project after [Kubernetes](http://kubernetes.io/).
+
+Our recent work has focused on delivering a stable API and user interface, marked by version 1.0 of Prometheus.
+We’re thrilled to announce that we’ve reached this goal, and [Prometheus 1.0 is available today](https://github.com/prometheus/prometheus/releases/tag/v1.0.0).
+
+
```
Comment (juliusv) [inline content/blog/2016-07-18-prometheus-1-0-released.md]: So many empty lines - such amaze

**Comment by juliusv** [comment]
:+1:

**Comment by fabxc** [comment]
It's markup, doesn't really matter :)
And isn't that the mandatory final line break?

**Comment by juliusv** [comment]
Ja, doesn't matter. Well, there are *two* final line breaks :) Anyways - yayyyy!

**Pull request merged**